### PR TITLE
Mention combining characters in the template

### LINKED ISSUE #103: Combining characters
https://w3c.github.io/bp-i18n-specdev/#char_ref_template

> For invisible characters (such as control characters) or for whitespace, the character and its surrounding bdi element should be omitted

I wonder if we should do the same for combining characters, otherwise it will look weird. Another approach is to use a `U+25CC DOTTED CIRCLE` before the combining character.

### PR DESCRIPTION
**Pull request by xfq**
Fix #103.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/bp-i18n-specdev/pull/104.html" title="Last updated on Jun 1, 2023, 12:30 PM UTC (2802f58)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/bp-i18n-specdev/104/9012381...2802f58.html" title="Last updated on Jun 1, 2023, 12:30 PM UTC (2802f58)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2187,6 +2187,15 @@ <h5>Character naming template</h5>
 &lt;span class="uname">U+XXXX Unicode_character_name&lt;/span>&lt;/span>
 </pre>
 
+<p>For combining characters (such as <span class="codepoint" translate="no"><bdi lang="en">&#x25cc;&#x0300;</bdi><span class="uname">U+0300 COMBINING ACCENT GRAVE</span></span>), if the specification uses a font which supports display with a <span class="codepoint" translate="no"><bdi lang="en">&#x25cc;</bdi><span class="uname">U+25CC DOTTED CIRCLE</span></span>, then a dotted circle should be used before the combining character:</p>
```
Comment (aphillips) [inline index.html:2190]: This is ambiguous? Does it mean "... uses a font which supports the dotted circle character with combining marks..."? How does a specification author know this? I think this is getting too complex and would rather not having folks flinging 25CCs about.

I think we should discuss automation of using an image? Or whether we can live with the combining mark showing without dotted circles?

**Comment by aphillips** [comment]
I think we should abandon this PR, since we now have guidance on using images.

**Comment by xfq** [comment]
The current text says "For invisible characters (such as control characters) or for whitespace, use an image instead of the character", but does not mention combining characters. I think we should add that.

By the way, I think whitespace are also "invisible", and some of them are also control characters.

**Comment by r12a** [comment]
The combining characters often work ok without the need for images.  If the font behaves well, it will typically add a dotted circle when displaying isolated combining marks.  So an image is only needed sometimes.

**Comment by aphillips** [comment]
Any update? Should we keep this or abandon?

**Comment by xfq** [comment]
I still think it's useful to mention combining characters in the template. Since there are merge conflicts, I will close this PR and craft a new one.

**Comment by xfq** [comment]
> The combining characters often work ok without the need for images. If the font behaves well, it will typically add a dotted circle when displaying isolated combining marks. So an image is only needed sometimes.

I found that the font didn't behave well on my system (without changing any font-related preferences), so it may appear for other users as well. Maybe we should use images?

-------

Screenshot:

Screenshot 2023-12-19 at 10 59 05

It's Chrome 120 on macOS 14.2. The rendered font seems to be Hiragino Sans.

**Pull request closed without merging**